docs/integration/frameworks/set-up-with-nextjs.md

```diff
@@ -0,0 +1,95 @@
+---
+description: A step-by-step guide to integrate Keyshade into your Next.js app for managing secrets and environment variables — no .env files needed.
+---
+
+# How to Set Up Keyshade into a Next.js Project
+
+## 1. Create a Next.js Project
+
+Create a new Next.js app and make sure you select TypeScript:
+
+```bash
+npx create-next-app my-app
+```
+You can find more info about creating a Next.js project [here](https://nextjs.org/docs/app/api-reference/cli/create-next-app).
+
+## 2. Install the Keyshade CLI
+Install globally via npm:
+```bash
+npm i -g @keyshade/cli
+```
+_Refer: [Installing the CLI](/docs/getting-started/installing-the-cli)_
+
+## 3. Set Up Your Keyshade Profile
+If you're using Keyshade **for the first time**, refer: [Setting up your profile](/docs/getting-started/setting-up-your-profile)
+
+If you've **used Keyshade before**, you can simply switch to an existing profile:
+
+```bash
+keyshade profile use my-profile
```

Comment:
instead of putting the user profile name here, I think you should add a placeholder, here is the convention 

1) Angle brackets: <my_key>
Meaning: Required value the user must replace.

2) Square brackets: [my_key]
Meaning: Optional value.

3) Curly braces: {my_key}
Meaning: Sometimes used in code examples to indicate interpolation or dynamic values.